distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
+
+[options="header"]
+|===
+|Operator |Component |Version
+
+|{TempoName}
+|Tempo
+|2.3.1
+
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
+|OpenTelemetry
+|0.93.0
+
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
+
+|===
+
+// Tempo section
+[id="distributed-tracing-rn_3-1_tempo-release-notes"]
+== {TempoName}
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+
+This update introduces the following enhancements for the {TempoShortName}:
+
+* Monolithic custom resource in dev-preview support.
+* Support for cluster-wide proxy environments.
+* Support for TraceQL to Gateway component.
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fixes for the {TempoShortName}:
+
+* Before this update, when a TempoStack was created with the `monitorTab` enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` ClusterRoleBinding was not created. This update resolves the issue by fixing the operator RBAC for the monitor tab when the operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
+* Before this update, when a TempoStack was created on an {product-title} cluster with only an IPV6 networking stack, the compactor and ingestor pods ran in the `CrashLoopBackOff` state, resulting in multiple errors. This update provides support for IPv6 clusters.(link:https://issues.redhat.com/browse/TRACING-3226[TRACING-3226])
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_known-issues"]
+=== Known issues
+
+//There is currently a known issue:
+There are currently known issues:
+
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+// Jaeger section
+[id="distributed-tracing-rn_3-1_jaeger-release-notes"]
+== {JaegerName}
+
+[id="distributed-tracing-rn_3-1_jaeger-release-notes_support-for-elasticsearch-operator"]
+=== Support for Elasticsearch Operator
```

Comment:
There is an attribute for the Elasticsearch `{es-op}` and Loki `{loki-op}` Operators now, please use them to maintain consistency